### ISSUE TITLE
3D printable case for MiniSynth Pi

### ISSUE THREAD

**Issue by pinkmouse101** [rsta2/minisynth#20]
Build finally finished, complete with 3d printed case. Thanks for all who created and assisted! :-)

I'm loath to put the case up as printable files, as this is a very personalized build, but if Rene wants to host the Fusion 360 file, I'm more than willing to post it here, that way others can tweak as desired.

cheers all
Al
![IMG_0751](https://user-images.githubusercontent.com/111195179/192852950-1681c91d-abff-4a3d-8811-2206f7f7c175.jpg)
![IMG_0752](https://user-images.githubusercontent.com/111195179/192852964-6502ad5a-d90d-4f5f-8f20-65c4073f9114.jpg)

**Comment by rsta2** [comment]
Wow, this looks great! If you want to contribute your construction CAD files, you can add another comment here with an archive file (e.g. .zip) included. Thanks!

**Comment by VasylMedvid77** [comment]
Wow. Just wow ))) (we talk about Owen Wilson grade "wow" here)
_In the next iteration I suggest to consider adding few rotary knobs for volume level, filter cuttoff, reverb amount levels_

**Comment by pinkmouse101** [comment]
Thanks, I'll post the files at the weekend with some more info.

**Comment by pinkmouse101** [comment]
Right, here we go. You will need the free, (or paid!), version of Fusion 360 from Autodesk to open this file. I'm deliberately not posting 3d printable files as my build used odd bits and pieces I had lying around, so any components you have will likely not fit this model, and I can't give links to buy particular parts I used, as many of them I forget where they came from! :-)

I don't have space to go into the Fusion editing process here, but there are many good tutorials out there, and I think I've done a reasonable job at naming sketches and layers to make things as simple as possible.

However, a few notes about stuff I did use. The touch panel I used was the waveshare 7" LCD C and after a few tribulations, it works well. I also used a USB B type connector for power, as it's a lot more robust than the Pi's onboard mini ones. That just runs to the 5v and GND pins on the header. The MIdi interface and DAC boards are not screwed in, I just used velcro, as during testing they went in and out a fair bit, however the Pi is mounted properly as it has a bit more weight to it.

You will need a selection of M3 screws, and I used heat set inserts for all the threaded holes, if you don't you'll need to reduce the size of the holes the screws thread into.

The design is sized to be printed on Ender 3 and up, go bigger and you'll need a bigger printer. ;-)

Well, enough waffle, here you go! Enjoy!

![Screenshot 2022-10-04 at 20 43 44](https://user-images.githubusercontent.com/111195179/193911290-3e01bb1e-d058-4049-8f1f-069d53a9b6a2.png)

[minisynth case v4.f3d.zip](https://github.com/rsta2/minisynth/files/9710081/minisynth.case.v4.f3d.zip)

Edit: Just a note, this is free to do with as you will, but  if you post it elsewhere a credit would be nice, (link back here will do), and if you make and sell lots of them, then I'm sure a local charity would appreciate a small donation. ;-)

**Comment by rsta2** [comment]
@pinkmouse101 Many thanks for providing the 3D printable files for your MiniSynth Pi case! I hope it will help MiniSynth Pi users to get such a great case, like it can be seen on your pictures.
